visual-tests/test-scripts/series/comparison.js

```diff
@@ -0,0 +1,72 @@
+(function(d3, fc) {
+    'use strict';
+
+    var data = [1, 2, 3].map(function() {
+        return fc.utilities.dataGenerator()
+            .seedDate(new Date(2014, 1, 1))
+            .generate(50);
+    });
+
+    var chartLayout = fc.utilities.chartLayout();
+    var chartBuilder = fc.utilities.chartBuilder(chartLayout);
+
+    d3.select('#comparison')
+        .call(chartBuilder);
+
+    // Create scale for x axis
+    var dateScale = fc.scale.dateTime()
+        .domain(fc.utilities.extent(data, 'date'))
+        .nice();
+
+    // Create scale for y axis
+    var priceScale = fc.scale.linear()
+        .domain(fc.utilities.extent(data, ['high', 'low']))
+        .nice();
+
+    // Create the axes
+    var dateAxis = d3.svg.axis()
+        .scale(dateScale)
+        .orient('bottom')
+        .ticks(5);
+
+    var priceAxis = d3.svg.axis()
+        .scale(priceScale)
+        .orient('left')
+        .ticks(5);
+
+    var percentageChange = fc.math.percentageChange()
+        .inputValue(function(d) { return d.close; })
+        .outputValue(function(d, value) { return (d.percentageChange = value); })
```

Comment:
It's doing both, but it doesn't need to. I think it's a side-effect of an earlier version of the test. I've fixed it and added a test for the other behaviour.